### DISCUSSION TITLE
how to debug database query in repl?

### DISCUSSION THREAD

**Discussion by ynl** [adonisjs/core#2101] [Help] [answered]
how to debug database query in repl?

**Comment by McSneaky** [comment, marked as answer]
You can set debug: true in db config files or start app with `DEBUG=knex:*` or for just queries `DEBUG=knex:query`

V5: https://preview.adonisjs.com/guides/database/debugging-queries
V4: https://adonisjs.com/docs/4.1/database#_debugging

EDIT: My bad, it was about `repl`. But should still work the same
